core/src/main/java/org/apache/accumulo/core/iterators/SortedKeyValueIterator.java

```diff
@@ -147,4 +147,25 @@
    *              if not supported.
    */
   SortedKeyValueIterator<K,V> deepCopy(IteratorEnvironment env);
+
+  /**
+   * Closes the Iterator. This must be overridden by the implementing class that has access to <tt>SortedKeyValueIterator source</tt> provided in the
+   * <tt>init</tt> method.
+   */
+  @Override
+  default void close() throws Exception {
```

Comment:
Not sure. IOException might make the most sense, or a custom one, SKVICloseException (for example... but not that, because that's awful).